Formula/envd.rb

```diff
@@ -29,12 +29,16 @@ def install
       -X github.com/tensorchord/envd/pkg/version.gitTreeState=clean
     ]
     system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/envd"
+    generate_completions_from_executable(bin/"envd", "completion", "--no-install",
+                                         shell_parameter_format: "--shell=",
+                                         shells:                 [:bash, :zsh])
   end
 
   test do
     output = shell_output("#{bin}/envd version --short")
     assert_equal "envd: v#{version}", output.strip
-    system bin/"envd", "init"
-    assert_predicate testpath/"build.envd", :exist?
+
+    stderr = shell_output("#{bin}/envd env list 2>&1 || true")
```

Comment:
15 can be whatever status code is expected.

```suggestion
    stderr = shell_output("#{bin}/envd env list 2>&1", 15)
```

Reply:
why 15 not 1? Running in a clean docker env and it gives me `1`

Reply:
Because I had to guess something and wanted to make it clear that it was a number.

Reply:
the ubuntu test image seems to have a different setup from others:
```
error: permission denied
```
Any suggestion on improving the test?